pkg/controllers/cluster/taint_manager.go

```diff
@@ -242,22 +235,25 @@ func (tc *NoExecuteTaintManager) syncClusterBindingEviction(key util.QueueKey) e
 		return err
 	}
 
+	var purgeMode policyv1alpha1.PurgeMode
+	if needEviction {
+		switch tc.NoExecuteTaintEvictionPurgeMode {
+		case "Gracefully":
+			purgeMode = policyv1alpha1.Graciously
+		case "Directly":
+			purgeMode = policyv1alpha1.Immediately
+		}
+	}
```

Comment:
Please remember to update it accordingly in  #6387.

Reply:
+1